boa/src/syntax/parser/statement/declaration/lexical_declaration.rs

```diff
@@ -0,0 +1,172 @@
+//! Lexical declaration parsing.
```

Comment:
Maybe we should call this just `lexical` module. It already lives in declaration module, so is implied that it's a declaration.